examples/objective-c-ios/Bugsnag Test App/ViewController.m

```diff
@@ -63,7 +63,8 @@ - (IBAction)generateSignal:(id)sender {
  This method causes a low-level exception from the operating system to terminate the app.  Upon reopening the app this signal should be notified to your Bugsnag dashboard.
  */
 - (IBAction)generateMachException:(id)sender {
-    void (*ptr)(void) = NULL;
+    // This should result in an EXC_BAD_ACCESS mach exception with code = KERN_INVALID_ADDRESS and subcode = 0xDEADBEEF
+    void (* ptr)(void) = (void *)0xDEADBEEF;
```

Comment:
The test case for mach exceptions should be updated to verify this behavior as well and assert that the recorded code and subcode equal the same values.

* [test](https://github.com/bugsnag/bugsnag-cocoa/blob/9660b880a97a57fbacb77b10a4b23b00b3104d56/features/unhandled_mach_exception.feature)
* [fixture](https://github.com/bugsnag/bugsnag-cocoa/blob/9660b880a97a57fbacb77b10a4b23b00b3104d56/features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/UnhandledMachExceptionScenario.m)

If the test server doesn't handle the unsigned 64-bit values correctly, then we should look at stringifying them before merging this down.

Reply:
The unhandled mach exception test case has been updated to verify the code and subcode, and has now passed successfully 👍